mods/ts/weapons/ballisticweapons.yaml

```diff
@@ -144,6 +144,7 @@ Grenade:
 		Palette: effect
 		BounceCount: 2
 	Warhead@1Dam: SpreadDamage
+		ValidTargets: Ground, Trees
```

Comment:
The weapons added with the latest push need a `ValidTargets: Ground, Trees` etc., so that you can force-fire trees.